scripts/publish.js

```diff
@@ -114,7 +115,7 @@ function copyReadme() {
 function copyScssFiles() {
   console.log('Copying SCSS files...');
   const onlyScssFiles = (input) => ['', '.scss'].includes(path.extname(input));
-  return fs.copy(`${libDir}/scss`, `${distTarget}/scss`, { filter: onlyScssFiles });
+  return fs.copy(`${coreLibDir}/scss`, `${distTarget}/scss`, { filter: onlyScssFiles });
```

Comment:
This change should (hopefully) only involve scss files. 

What happens in this step is that we copy all the scss to the top level directory of the distributed package, so it can be referenced like `@kirbydesign/designsystem/scss/utils` instead of `@kirbydesign/designsystem/src/scss/utils`. 

Before, all our scss utils etc. existed in the designsystem, but now we have to copy it from core, as everything has been moved there. But we still want to preserve backwards compatibility, so it should come bundled with the designsystem package :)